Date/Span.php

```diff
@@ -1047,7 +1047,7 @@ function lower($time)
      * @access   public
      * @static
      */
-    function compare($time1, $time2)
+    static function compare($time1, $time2)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.